package.json

```diff
@@ -51,7 +51,12 @@
   "module": "lib.esm.js",
   "types": "index.d.ts",
   "dependencies": {
+    "@date-io/date-fns": "^1.3.13",
+    "@date-io/moment": "^1.3.13",
```

Comment:
These are packages for Date Picker, we are building a time picker so this is not required here.